politeiawww/dcc.go

```diff
@@ -899,3 +910,139 @@ func (p *politeiawww) getDCCComments(token string) ([]www.Comment, error) {
 
 	return comments, nil
 }
+
+func (p *politeiawww) processSetDCCStatus(ad cms.SetDCCStatus, u *user.User) (*cms.SetDCCStatusReply, error) {
```

Comment:
You're missing some validation here.

* The pubkey in the request needs to match the admins active identity.
* Is a Reason required?  If so, you need to check that.

Also, the variable `ad` seems to be left over from when it was `cms.ApproveDCC`.  Probably want to update that.